rafthttp/transport.go

```diff
@@ -180,7 +180,10 @@ func (t *Transport) Send(msgs []raftpb.Message) {
 		}
 		to := types.ID(m.To)
 
+		t.mu.RLock()
 		p, ok := t.peers[to]
+		t.mu.RUnlock()
```

Comment:
You can clean this up in the near future. I am trying to clean up the race detected by the race detector in the commit. Or you want me to go over the code?